examples-nosql-dotnet-sdk/sqlexamples/TableJoins.cs

```diff
@@ -0,0 +1,171 @@
+/*-
+ * Copyright (c) 2018, 2023 Oracle and/or its affiliates. All rights reserved.
+ *
```

Comment:
If this is a new file, then the copyright should not be 2019, 2023.    Since we have 2024 around the corner you can make it 2023, 2024.